docs/package_templates/autotools_package/all/conanfile.py

```diff
@@ -22,13 +22,13 @@ class PackageConan(ConanFile):
     name = "package"
     description = "short description"
     # Use short name only, conform to SPDX License List: https://spdx.org/licenses/
-    # In case not listed there, use "LicenseRef-<license-file-name>"
+    # In case not listed there, use "DocumentRef-<license-file-name>:LicenseRef-<package-name>"
```

Comment:
The `DocumentRef` format has been recommended in all recent PRs where a non-standard license has been used.